src/BloomBrowserUI/bookEdit/js/textOverPicture.ts

```diff
@@ -368,7 +386,19 @@ export class TextOverPictureManager {
                 );
             } else {
                 // Not currently dragging
-                if (Comical.getBubbleHit(container, ev.offsetX, ev.offsetY)) {
+                const targetElement = ev.target as HTMLElement;
+                const targetBounds = targetElement.getBoundingClientRect();
+
+                // These coordinates need to be relative to the canvas (which is the same as relative to the image container).
+                const [targetX, targetY] = this.getCoordinatesRelativeTo(
+                    ev.offsetX,
+                    ev.offsetY,
+                    targetBounds,
+                    containerBounds,
+                    styleInfo
+                );
+
+                if (Comical.getBubbleHit(container, targetX, targetY)) {
```

Comment:
Seems there's a chunk of common code that could be extracted from this and mouseDown? Or, would it be much more expensive to just pass targetElement and container to getCoordinatesRelativeTo()? Let it do the whole job of converting an offset relative to one element to one relative to the other.